<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>551</width>
    <height>561</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Admin - KB Roadways</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../__Assets/Logos/CompanyLogo.png</normaloff>../__Assets/Logos/CompanyLogo.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lblLogo">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>20</y>
      <width>121</width>
      <height>91</height>
     </rect>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../__Assets/Logos/CompanyLogo.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnUpdateTransports">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>130</y>
      <width>231</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Update Transports</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAddTransport">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>130</y>
      <width>231</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Add Transport</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAddCompany">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>200</y>
      <width>231</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Add Company</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnUpdateCompanies">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>200</y>
      <width>231</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Update Companies</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAddVehicleOwner">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>270</y>
      <width>231</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Add Vehicle Owner</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnUpdateVehicleOwners">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>270</y>
      <width>231</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Update Vehicle Owners</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnPaymentStatus">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>480</y>
      <width>231</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Payment Status</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnBillSubmission">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>480</y>
      <width>231</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Bill Submission</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAddDriver">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>340</y>
      <width>231</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Add Driver</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnUpdateDrivers">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>340</y>
      <width>231</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Update Drivers</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnUpdateCars">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>410</y>
      <width>231</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Update Cars</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAddCar">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>410</y>
      <width>231</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Add Car</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
